src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -430,28 +437,30 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
           datasetToUpdate.items.length === 0 ? (
             <NoItemsPlaceholder></NoItemsPlaceholder>
           ) : (
-            <BaseTable
-              classPrefix="DatasetEditorTable"
-              data={tableData}
-              height={TABLE_HEIGHT}
-              width={defaultColumns.reduce((totalWidth, column) => {
-                return totalWidth + column.width
-              }, 0)}
-              rowHeight={ROW_HEIGHT}
-              overscanRowCount={5}
-              columns={defaultColumns}
-              rowClassName={'DatasetEditorRow'}
-              rowProps={({ rowData }) => {
-                return {
-                  'aria-selected': rowData.isSelected,
-                }
-              }}
-              headerCellProps={{
-                role: 'columnheader',
-              }}
-              onRowsRendered={rebuildTooltip}
-              onScroll={rebuildTooltip}
-            ></BaseTable>
+            <AutoResizer height={TABLE_HEIGHT} onResize={rebuildTooltip}>
+              {({ height, width }: { height: number; width: number }) => (
+                <BaseTable
+                  classPrefix="DatasetEditorTable"
+                  data={tableData}
+                  height={height}
+                  width={width > totalColumnWidth ? width : totalColumnWidth}
```

Comment:
If the container width is greater than the sum of all column widths, use the container width.